qutebrowser/browser/commands.py

```diff
@@ -523,6 +523,39 @@ def tab_detach(self):
         cur_widget = self._current_widget()
         self._tabbed_browser.close_tab(cur_widget, add_undo=False)
 
+    @cmdutils.register(instance='command-dispatcher', scope='window')
+    @cmdutils.argument('index', completion=miscmodels.buffer)
+    def tab_take(self, index):
+        """Take a tab from another window.
+
+        Args:
+            index: The [win_id/]index of the tab to take. Or a substring
+                   in which case the closest match will be focused.
+        """
+        tabbed_browser, tab = self._resolve_buffer_index(index)
+
+        if tabbed_browser == self._tabbed_browser:
+            raise cmdexc.CommandError("Can't take a tab from the same window")
+
+        self._open(tab.url(), tab=True)
+        tabbed_browser.close_tab(tab, add_undo=False)
+
+    @cmdutils.register(instance='command-dispatcher', scope='window')
+    @cmdutils.argument('win_id', completion=miscmodels.window)
+    def tab_give(self, win_id: int):
```

Comment:
Looking back at the discussion in #1788, wasn't the intention for `:tab-give` without an ID to replace `:tab-detach`?

Reply:
Ah that's interesting, I never thought about it that way personally. I assumed we were going to keep all 3, but in my previous PR I did end up implementing `:tab-detach` mainly using `:tab-give`.

Should I add an alias for `:tab-detach` to call `:tab-give` by itself?